src/Jobs/Recruit.Vacancies.Jobs/appSettings.Development.json

```diff
@@ -29,5 +29,12 @@
   "Slack": {
     "WebHookUrl": ""
   },
-  "UseTableStorageQueryStore": true
+  "UseTableStorageQueryStore": true,
+  "ProviderRelationshipsApiConfiguration": {
+    "ApiBaseUrl":"",
+    "ClientId":"",
```

Comment:
We don't need it in this appSettings.Development.json file since you have it in the appSetting.json file. Assuming we will have it in our secrets file otherwise how did you get it to work.

Also where are the ARM 💪 template changes?

Reply:
Yes, we will use the secrets to manage this. I have removed it and also will follow up with arm template changes.

Reply:
Added azure resource changes.